install/deploy_management_nodes.md

```diff
@@ -725,6 +725,37 @@ The LiveCD needs to authenticate with the cluster to facilitate the rest of the
 
 After the NCNs are booted, the BGP peers will need to be checked and updated if the neighbor IP addresses are incorrect on the switches. Follow the steps below and see [Update BGP Neighbors](../operations/network/metallb_bgp/Update_BGP_Neighbors.md) for more details on the BGP configuration.
 
+**`IMPORTANT:`** If the management switches are using CANU generated 1.0 (preconfig) this procedure should be skipped.
+
+* To check if the management switches are using the 1.0 (Preconfig) log into both spine switches and see if a `motd banner` exists and looks similar to the examples below.
```

Comment:
This looks like it should be a step. For example...

---

1. Check if the management switches are using the 1.0 (preconfig) by logging into both spine switches and seeing if a `motd banner` exists.
    
    The CSM version must be `1.0`, and the CANU version should not matter. An accurate `motd banner` for Mellanox and Aruba will look similar to the following examples:

    Mellanox example:
    
     ```
     example
     ```
    Aruba example:
    
     ```
     example
     ```